src/EditorFeatures/TestUtilities/Workspaces/TestWorkspace.cs

```diff
@@ -193,6 +194,7 @@ private static IList<Exception> Flatten(ICollection<Exception> exceptions)
             var aggregate = new AggregateException(exceptions);
             return aggregate.Flatten().InnerExceptions
                 .Select(UnwrapException)
+                .Where(ex => !(ex is JoinableTaskContextException))
```

Comment:
The right fix here is to reconcile how we construct WPF text views with what JTF expects.  I'm currently undecided on whether this should block this PR.  If we care about inserting we may want to merge this and file a bug to resolve later.